src/store/membership_fee/actions.ts

```diff
@@ -88,7 +94,7 @@ export const actions = {
 	},
 	[ setFee ]( context: ActionContext<MembershipFee, any>, payload: GenericValuePayload ): Promise<void> {
 		context.commit( SET_FEE, payload.selectedValue );
-		context.commit( SET_FEE_VALIDITY, validateFeeOnClientSide( payload.selectedValue ) );
+		context.commit( SET_FEE_VALIDITY, validateFeeOnClientSide( context, payload.selectedValue ) );
```

Comment:
The `setFee` unit tests need a test case that shows that the fee will be reset to empty on invalid